### PR TITLE
docs: Added the steps to develop fiddle.

### PR DESCRIPTION
**Pull request by arnitkun**
Added the steps to run fiddle, as mentioned in #259

### CODE REVIEW

**Comment by malept** [comment]
Wouldn't the steps be "Download the binary and run"? Installing it from the repo is only if you want to work on Fiddle itself.

**Comment by arnitkun** [comment]
yes, I was only thinking from the point of view of someone who wants to develop himself, but then he/she should know how to do this. 

Adding the instruction for "end-users".

**Comment by malept** [comment]
In that case, the "how to download Fiddle to develop on it" instructions should be in a `CONTRIBUTING.md` file (which I'm kind of surprised that it's not in this repo yet) instead of `README.md`.

File: README.md [hunk 1]
```diff
@@ -56,13 +56,17 @@ fiddle has grown up, export it as a project with or without
...
 
 1. Clone this repository: `git clone https://github.com/electron/fiddle.git`.
 2. Change current directory to fiddle: `cd ./fiddle`
 3. Run `npm install`
 4. Run `npm start`
 
+### Running Fiddle
+
+Download and run the latest binary from the electron [releases page](https://electronjs.org/releases/stable).
+
```
Comment (malept) [inline README.md]: I don't think this is necessary, since there's already a Download link after the first paragraph in the README.

Comment (arnitkun) [reply README.md]: okay, removing this and placing the build instructions in contributing.md. 

One quick question, should I put a generic contribution guide or would one/more of the members do it?

Comment (felixrieseberg) [reply README.md]: I think a contributor's guide is a wonderful idea!

**Comment by felixrieseberg** [comment]
There hasn't been much movement here in a while, so I'm closing this for now. We could always reopen it!

**Comment by arnitkun** [comment]
Yeah, Apologies for that, very busy right now.

**Pull request closed without merging**